### PR TITLE
Get rid of mix of languages

### LINKED ISSUE #19: Fix language uniformity
We have some output messages in russian language: https://github.com/tolstoyevsky/hubot-happy-birthder/blob/2f609650767d971764f2f9fda904861948767223/src/birthder.js#L209 and some in english: https://github.com/tolstoyevsky/hubot-happy-birthder/blob/2f609650767d971764f2f9fda904861948767223/src/birthder.js#L39.
So I suggest to choose one default language and fix messages text.

### PR DESCRIPTION
**Pull request by eugulixes**
Fixes #19.

### CODE REVIEW

**Review by sibiryoff — APPROVED** [review]

File: src/birthder.js [hunk 1]
```diff
@@ -245,10 +243,10 @@
             let date = msg.match[2], users;
             users = findUsersBornOnDate(moment(date, DATE_FORMAT), robot.brain.data.users);
             if (users.length === 0) {
-                return msg.send(MSG_UNABLE_TO_LOCATE_USERS);
+                return msg.send("Could not find any user with the specified birthday.");
             }
             let userNames = users.map(user => `@${user.name}`), message;
-            message = `*${date}* день рождения у ${userNames.join(', ')}\n${quote()}`;
```
Comment (Sergei-vb) [inline src/birthder.js]: Why was remove this information?

File: src/birthder.js [hunk 2]
```diff
@@ -245,10 +243,10 @@
...
+            message = `${userNames.join(', ')}`;
```
Comment (TheSergeX) [inline src/birthder.js:250]: Why was remove this information? Isn't this mistake?

Comment (eugulixes) [reply src/birthder.js:250]: It was a bug which I silently fixed.

Comment (TheSergeX) [reply src/birthder.js:250]: ok

**Review by TheSergeX — APPROVED** [review]

**Pull request merged**